e2e-environment/Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG RAIDEN_VERSION="v2.1.0"
-ARG CONTRACTS_PACKAGE_VERSION="v0.40.0"
-ARG CONTRACTS_VERSION="0.40.0"
-ARG SERVICES_VERSION="v1.0.0"
+ARG RAIDEN_VERSION="arbitrum"
+ARG CONTRACTS_PACKAGE_VERSION="v1.0.0rc2"
```

Comment:
There's an RC3 now, maybe it's a good idea to update

Reply:
Is there? What does it include? The second release candidate includes all relevant changes that are not release procedure related, right?
Note that this will change anyway soon...

Reply:
It includes the version bump, but it's fine to move on with this one then.

Reply:
✔️